### PR TITLE
Imporovements to exam schedule  pagination component

### LINKED ISSUE #56: Improve Table Pagination Actions in Schedule Page to match the updated UI Design
Let's change the table pagination actions to match the updated UI design.

- [x] Add rows per page option
- [x] Remove page number buttons as they are not present in the updated UI design
- [x] Add navbar
- [x] Align the last column to the right to remove unwanted whitespaces on the right side
- [x] Change page title

Screenshot (14)
Screenshot (15)

### PR DESCRIPTION
**Pull request by samadhi-k**
image

### CODE REVIEW

File: src/pages/schedule-page.js
Comment (Malith-19) [inline src/pages/schedule-page.js:1]: Let's add the clickable option to each row and it should navigate to the relevant exam monitoring page of that selected exam. 

The only change need to do is to show the hover when the pointer goes to the row.

Comment (Malith-19) [inline src/pages/schedule-page.js:1]: Can we center the table footer which has showing the page number and other stuff.

**Review by Malith-19 — CHANGES_REQUESTED** [review]
Add some suggestions.

**Comment by Malith-19** [comment]
There are some conflicts with the main please check it too.

**Comment by samadhi-k** [comment]
image
Added the requested changes

**Review by Malith-19 — APPROVED** [review]

**Pull request merged**